### PR TITLE
WAV-2592: API to fetch sim total usages: Manually update the external api doc

### PR DESCRIPTION
**Pull request by lizozavr**

### CODE REVIEW

File: docs/api/sim-card-usage.md [hunk 1]
```diff
@@ -250,4 +255,51 @@ Call usage for a given **month** in a given **year** can be accessed by using th
...
+Keep in mind that the response is given as a list of usages and not returned as a regular top level JSON object.
+
+## Total usage for a SIM card
+
+Ger the total data usage of a specific SIM Card for a given **period** can be accessed by using the following URL:
+
+`POST /simcardexternalapi/simcardusage`
+
+**Request body structure and types**
+Request body contains GetSimCardUsageDTO with such fields:
```
Comment (clissau) [inline docs/api/sim-card-usage.md:268]: Add blank line between header and body.

File: docs/api/sim-card-usage.md [hunk 2]
```diff
@@ -17,25 +19,27 @@ When querying it is also required to specify which SIM card the usage should be
...
+|-------------|-------------------|---------------------------------------------------------------|
 | icc         | String            | The unique identifier for the SIM card generating the usage.  |
-| year        | Number            | Year is given in the ISO 8601 format for example 2019.             |
+| year        | Number            | Year is given in the ISO 8601 format for example 2019.        |
 | month       | Number            | Month is represented as a number for example 'april = 4'      |
 | usageDate   | ISO 8601 DateTime | In UTC time.                                                  |
 | bytesIn     | Number            |                                                               |
 | bytesOut    | Number            |                                                               |
 | sessionId   | String            | Unique identifier for the session on the network.             |
 | imei        | String            | The observed IMEI number for the device generating the usage. |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.

File: docs/api/sim-card-usage.md [hunk 3]
```diff
@@ -64,32 +68,33 @@ Data usage for a given **month** in a given **year** can be accessed by using th
...
-| month           | Number            | Month is represented as a number for example 'april = 4'        |
-| usageDate       | ISO 8601 DateTime | In UTC time.                                                    |
-| caller          | String            | Regular string and not necessarily a valid number.              |
-| called          | String            | E164 MSISDN, e.g. *+4593709603*                                 |
-| fromCountryCode | String            | Alpha-2 code                                                    |
-| toCountryCode   | String            | Alpha-2 code                                                    |
-| direction       | [Directions](/general-information/data-types#directions)            |               |
+| Field           | Type                                                     | Description                                                  |
+|-----------------|----------------------------------------------------------|--------------------------------------------------------------|
+| icc             | String                                                   | The unique identifier for the SIM card generating the usage. |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.

File: docs/api/sim-card-usage.md [hunk 4]
```diff
@@ -139,17 +144,17 @@ SMPP usage for a given **month** in a given **year** can be accessed by using th
...
-| month           | Number            | Month is represented as a number for example 'april = 4'        |
-| usageDate       | ISO 8601 DateTime | In UTC time.                                                    |
-| caller          | String            | Regular string and not necessarily a valid number.              |
-| called          | String            | Regular string and not necessarily a valid number.              |
-| fromCountryCode | String            | Alpha-2 code                                                     |
-| toCountryCode   | String            | Alpha-2 code                                                     |
-| direction       | [Directions](/general-information/data-types#directions)            | |
+| Field           | Type                                                     | Description                                                  |
+|-----------------|----------------------------------------------------------|--------------------------------------------------------------|
+| icc             | String                                                   | The unique identifier for the SIM card generating the usage. |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.

File: docs/api/sim-card-usage.md [hunk 5]
```diff
@@ -195,28 +200,28 @@ SMPP usage for a given **month** in a given **year** can be accessed by using th
...
-| fromCountryCode | String            | Alpha-2 code                                                                                                                  |
-| toCountryCode   | String            | Alpha-2 code                                                                                                                  |
-| direction       | [Directions](/general-information/data-types#directions)            |                                 |
+| Field           | Type                                                     | Description                                                                                                                  |
+|-----------------|----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| icc             | String                                                   | The unique identifier for the SIM card generating the usage.                                                                 |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.

File: docs/api/sim-card-usage.md [hunk 6]
```diff
@@ -195,28 +200,28 @@ SMPP usage for a given **month** in a given **year** can be accessed by using th
...
+| month           | Number                                                   | Month is represented as a number for example 'april = 4'                                                                     |
+| usageDate       | ISO 8601 DateTime                                        | In UTC time.                                                                                                                 |
+| caller          | String                                                   | E164 MSISDN, e.g. *+4593709603*                                                                                              |
+| called          | String                                                   | E164 MSISDN, e.g. *+4593709603*                                                                                              |
+| duration        | Number                                                   | Duration of the call measured in seconds, the result disregards network connection time. i.e. Only the billed call duration. |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.

File: docs/api/sim-card-usage.md [hunk 7]
```diff
@@ -250,4 +255,51 @@ Call usage for a given **month** in a given **year** can be accessed by using th
 ```
 
 !!! tip
-    Keep in mind that the response is given as a list of usages and not returned as a regular top level JSON object.
\ No newline at end of file
+Keep in mind that the response is given as a list of usages and not returned as a regular top level JSON object.
+
+## Total usage for a SIM card
+
+Ger the total data usage of a specific SIM Card for a given **period** can be accessed by using the following URL:
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Typo in `Ger`.
`Card` is previously written in small letters.

File: docs/api/sim-card-usage.md [hunk 8]
```diff
@@ -250,4 +255,51 @@ Call usage for a given **month** in a given **year** can be accessed by using th
...
+`POST /simcardexternalapi/simcardusage`
+
+**Request body structure and types**
+Request body contains GetSimCardUsageDTO with such fields:
+
+| Field         | Type          | Description                                              |
+|---------------|---------------|----------------------------------------------------------|
+| iccs          | List<String>  | The list of unique identifiers of the SIM cards          |
+| startDateTime | LocalDateTime | Start date and time of period for getting sim card usage |
+| endDateTime   | LocalDateTime | End date and time of period for getting sim card usage   |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Please write `SIM` capitalized.

File: docs/api/sim-card-usage.md [hunk 9]
```diff
@@ -250,4 +255,51 @@ Call usage for a given **month** in a given **year** can be accessed by using th
...
+    "endDateTime" : "2021-08-01T00:00:01.715"
+  }
+```
+
+**Response structure and types**
+
+| Field         | Type   | Description                                                 |
+|---------------|--------|-------------------------------------------------------------|
+| icc           | String | The unique identifier for the SIM card generating the usage |
+| sumTotalBytes | Long   | The total data usage sum of bytes of sim card               |
```
Comment (clissau) [inline docs/api/sim-card-usage.md]: Please write `SIM` capitalized.

**Review by clissau — CHANGES_REQUESTED** [review]
Minor changes requested.

File: docs/api/sim-card-usage.md [hunk 10]
```diff
@@ -250,4 +255,51 @@ Call usage for a given **month** in a given **year** can be accessed by using th
...
+
+`POST /simcardexternalapi/simcardusage`
+
+**Request body structure and types**
+Request body contains GetSimCardUsageDTO with such fields:
+
+| Field         | Type          | Description                                              |
+|---------------|---------------|----------------------------------------------------------|
+| iccs          | List<String>  | The list of unique identifiers of the SIM cards          |
+| startDateTime | LocalDateTime | Start date and time of period for getting sim card usage |
```
Comment (haodengwavely) [inline docs/api/sim-card-usage.md]: LocalDateTime is java data type. It should be a String in json request.

**Review by haodengwavely — CHANGES_REQUESTED** [review]

**Review by haodengwavely — APPROVED** [review]

**Review by clissau — APPROVED** [review]

**Pull request merged**